src/main/webapp/js/userMapData.json

```diff
@@ -772,6 +821,7 @@
 ["Worcester Polytechnic Institute","USA"],
 ["Wright State University","USA"],
 ["WSU Vancouver","USA"],
+["Xavier University","USA"],
 ["Xavier University of Louisiana","USA"],
```

Comment:
These two institutes are different.

`Xavier University` is based in Cincinnati, Ohio;
`Xavier University of Louisiana` is based in New Orleans, Louisiana.

Both entries kept.